src/mumble/Log.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>446</width>
-    <height>334</height>
+    <width>524</width>
```

Comment:
Yeah. That was qtcreator's doing when I saved it. I remove that if you want. It was buried under other commits and I was lazy....